src/test/java/core/basesyntax/service/impl/CsvReaderServiceTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.DataReaderService;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CsvReaderServiceTest {
+    private static final String INVALID_PATH = "src/test/resources/dummy-path/dummy.csv";
```

Comment:
```suggestion
    private static final String INVALID_PATH = "src/test/resources/non/dexistent/path/dummy.csv";
```
dummy doesn't mean non exisent